cloudera-on-centos/azuredeploy.json

```diff
@@ -25,24 +25,41 @@
     "cmPassword": {
       "type": "securestring",
       "metadata": {
-        "description": "password for the Cloudera Manager"
+        "description": "password for the Cloudera Manager (A mix of upper and lower-case characters, digits and symbols)"
       }
     },
     "storageAccountPrefix": {
       "type": "string",
       "metadata": {
-        "description": "Unique namespace for the Storage Account where the Virtual Machine's disks will be placed"
+        "description": "Unique namespace for the Storage Account where the VM disks will be placed (alphahumeric characters only)"
       }
     },
     "dnsNamePrefix": {
       "type": "string",
       "metadata": {
-        "description": "Unique public dns name where the Virtual Machines will be exposed"
+        "description": "Unique public DNS name where the VMs will be exposed"
       }
     },
     "region": {
       "type": "string",
-      "defaultValue": "japanwest",
+      "defaultValue": "westeurope",
```

Comment:
that's a moving target. Should we have to keep updating the template as it rolls out?
Suggestions are welcome.